documentation/docs/client/setup/initializing_sdk.md

```diff
@@ -4,26 +4,32 @@ title: Initializing SDK
 sidebar_position: 2
 ---
 
- As a first step, you need to initialize `ChatClient`, which is the main entry point for all operations in the library. You should only create the client once and re-use it across your application. Typically `ChatClient` is initialized in `Application` class:
+Integrating with the you need to initialize the `ChatClient`. The `ChatClient` is the main entry point for all operations in the library. Create a single `ChatClient` and re-use it across your application.
```

Comment:
> Integrating with the you

Not sure what happened here

Reply:
Typos, typos happened 😞 . Let me fix that.